velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -253,6 +253,7 @@ class DecodedVectorTest : public testing::Test, public VectorTestBase {
     }
   }
 
+  // Three levels of encoding: Dict(Constant(ComplexVector))
```

Comment:
This comment is incorrect. There are only 2 levels of encoding: Dict(Constant). ComplexVector represents a type, not encoding.